src/Umbraco.Core/Services/SectionService.cs

```diff
@@ -212,7 +212,7 @@ public Section GetByAlias(string appAlias)
         public void MakeNew(string name, string alias, string icon)
         {
             var sections = GetSections();
-            var nextSortOrder = sections != null ? sections.Max(x => x.SortOrder) + 1 : 1;
+            var nextSortOrder = sections != null && sections.Any() ? sections.Max(x => x.SortOrder) + 1 : 1;

```

Comment:
I don't recall finding that, but the change I've made here isn't actually adding the null check, it's adding the "is empty" check.  If I remember right from a unit test I found that a non-null but empty list of sections, errored when you tried to call `Max()` on it.  Hence the check now for "is not null and is not empty".